packages/@lwc/template-compiler/src/__tests__/fixtures/expression-complex/valid/computed-property/actual.html

```diff
@@ -0,0 +1,3 @@
+<template>
+    <section onclick={bar.arr[baz]}>{bar.arr[baz]}</section>
```

Comment:
Does this work correctly with deep member expressions (`bar.baz.arr[quux]`)? How about member expressions inside the computed part (`bar.arr[baz.quux]`)?

Reply:
Added tests for those examples, thanks